packages/story-editor/src/components/panels/design/textAccessibility/textAccessibility.js

```diff
@@ -0,0 +1,167 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import { __, TranslateWithMarkup } from '@googleforcreators/i18n';
+import {
+  DropDown,
+  Text,
+  THEME_CONSTANTS,
+} from '@googleforcreators/design-system';
+import { getTextElementTagNames } from '@googleforcreators/output';
+import { useFeature } from 'flagged';
+import { useState, useEffect } from '@googleforcreators/react';
+
+/**
+ * Internal dependencies
+ */
+import { Row } from '../../../form';
+import { SimplePanel } from '../../panel';
+import { useHighlights, states, styles } from '../../../../app/highlights';
+import { MULTIPLE_VALUE, MULTIPLE_DISPLAY_VALUE } from '../../../../constants';
+import { combineElementsWithTags } from './utils';
+
+// Options to be mapped for the dropdown
+const semanticHeadingsOptionsMap = [
+  {
+    label: __('Heading 1', 'web-stories'),
+    value: 'h1',
+  },
+  {
+    label: __('Heading 2', 'web-stories'),
+    value: 'h2',
+  },
+  {
+    label: __('Heading 3', 'web-stories'),
+    value: 'h3',
+  },
+  {
+    label: __('Paragraph', 'web-stories'),
+    value: 'p',
+  },
+  {
+    label: __('Automatic', 'web-stories'),
+    value: 'auto',
+  },
```

Comment:
Let's move this to the beginning of the list since this is suppose to be the default behavior.

Reply:
Done in https://github.com/GoogleForCreators/web-stories-wp/pull/11888/commits/6dc71c5d492413303d5c952fe5ccfef660dcb476